src/engine/strat_engine/blockdev.rs

```diff
@@ -97,7 +98,23 @@ pub fn find_all() -> EngineResult<BTreeMap<PoolUuid, BTreeMap<DevUuid, BlockDev>
     Ok(pool_map)
 }
 
+/// Zero sectors at the given offset
+pub fn wipe_sectors(path: &PathBuf, offset: &Sectors, sector_count: &Sectors) -> EngineResult<()> {
```

Comment:
There's currently no point in using references to Sectors for parameter types, since they are implicitly Copy.